tutorials/crictl.md

```diff
@@ -1,28 +1,35 @@
 # CRI-O crictl Tutorial
+<!-- markdownlint-enable MD013 -->
```

Comment:
Just curious; Is the `MD013` rule enabled by default? If so, why the need to explicitly enable it at the beginning of the file?

Reply:
It is, I think I placed it there to get something else on my clipboard en forgot it, I'll fix it (maybe tomorrow).